src/pybind/mgr/cephadm/module.py

```diff
@@ -378,6 +376,8 @@ def __init__(self, *args, **kwargs):
             self.inventory = dict()
         self.log.debug('Loaded inventory %s' % self.inventory)
 
+        self.to_remove_osds = set()  # TODO: load them from mon store
```

Comment:
In case we load them from the mon store, we really need a way to cancel the op.

Reply:
If we want to allow cancellation we probably have set-up two different lists. `in_queue` and `started_removal` while only OSDs in `in_queue` should be able to get canceled. OSDs that are already in the middle of getting destroyed will leave a lot of debris behind I guess..

Reply:
I'm not saving this persistently for now (neither in cephadm nor in osd_support) for now..
That's something for later and needs some more review